core/src/main/java/org/hisp/dhis/android/core/category/CategoryCategoryComboLinkStoreImpl.java

```diff
@@ -46,14 +64,48 @@ public long insert(CategoryCategoryComboLinkModel entity) {
         return executeInsert();
     }
 
+    @Override
+    public int delete() {
+        return databaseAdapter.delete(CategoryCategoryComboLinkModel.TABLE);
+    }
+
+    @Override
+    public boolean delete(@NonNull CategoryCategoryComboLinkModel entity) {
+
+        validate(entity);
+
+        bind(deleteStatement, entity);
+
+        return execute(deleteStatement);
+    }
+
+    @Override
+    public boolean update(@NonNull CategoryCategoryComboLinkModel oldCategoryCategoryComboLinkMode,
```

Comment:
To update one row only is necessary one model, you are pass as argument new and old, can you refactor this method?

Reply:
This case is similar to the one that I described you before all the link tables have two values and at the same time they are the primary key, with only one model, I'll lose the where clause.